robot/EDA/resources/EDA.py

```diff
@@ -400,6 +406,13 @@ def go_to_custom_settings_setup(self):
         self.selenium.go_to(url)
         self.salesforce.wait_until_loading_is_complete()
 
+    def go_to_new_eda_settings(self):
+        """ Navigates to the new EDA Settings Page"""
+        url = self.cumulusci.org.lightning_base_url
+        url = "{}/lightning/n/New_EDA_Settings_QA_Only".format(url)
+        self.selenium.go_to(url)
```

Comment:
@rhakeshj are the devs providing you with a QA only url? If so will this be available as part of the managed package when you do a release?

Reply:
This is a temporary URL made available for QE purpose. Before merging it to main branch this will be updated accordingly